lib/routes/token-codes.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const errors = require('../error')
+const isA = require('joi')
+const validators = require('./validators')
+const HEX_STRING = validators.HEX_STRING
+
+module.exports = (log, db, customs) => {
+  return [
+    {
+      method: 'POST',
+      path: '/tokenCodes/verify',
+      config: {
+        validate: {
+          payload: {
+            uid: isA.string().max(32).regex(HEX_STRING).required(),
+            code: isA.string().required() // code length is configurable, don't set min/max
```

Comment:
I think it would be reasonable to enforce a minimum length here as an extra layer of defense against config errors, even if it's lower than the default config value.

Reply:
Also, a very large max may be a good idea just as a safety valve against DoS attempts, even it's like 1024 characters or something.